### ISSUE TITLE
glabels and dependencies with invalid key (Bugzilla Bug 514)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#514]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 514
Date: 2014-02-18 10:40:43 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;
To:   OpenMandriva QA &lt;<bugs@openmandriva.org>&gt;
CC:   siriustheking@yahoo.com, @tpgxyz

Duplicates:   #344
Depends on:   #344
Last updated: 2014-02-25 08:51:45 +0000

## Comment 3254
Date: 2014-02-18 10:40:43 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

Trying to install Glabels I found several invalid key in packages involved:

1) glabels-2.3.0-4-omv2013.0.x86_64.rpm: Invalid Key ID (OK (DSA/SHA1, Sun 21 Apr 2013 00:24:17 CEST, Key ID 47a477b1f192ecfb))

2) lib64glabels3.0_7-2.3.0-4-omv2013.0.x86_64.rpm: Invalid Key ID (OK (DSA/SHA1, Sun 21 Apr 2013 00:24:20 CEST, Key ID 47a477b1f192ecfb))

3) lib64iec16022_0-0.2.4-2-omv2013.0.x86_64.rpm: Invalid Key ID (OK (DSA/SHA1, Sat 20 Apr 2013 12:13:21 CEST, Key ID 52f5e855200d971d))

4) lib64zint2.4-2.4.3-1-omv2013.0.x86_64.rpm: Invalid Key ID (OK (DSA/SHA1, Sat 20 Apr 2013 00:32:45 CEST, Key ID 5adea9cfeb9f0720))

## Comment 3283
Date: 2014-02-24 21:56:06 +0000
From: Gus Ballan &lt;<siriustheking@yahoo.com>&gt;

I think that this should be added to bug #344.

## Comment 3286
Date: 2014-02-24 22:02:12 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

It depends on OMLx team I think.
If they correct all invalid keys simultaneously it make sense to open only one bug for all packages.

If they correct a single package it's better to have a bug for every package.

## Comment 3291
Date: 2014-02-25 08:51:45 +0000
From: @tpgxyz

Hi,

yes we know about this issue.

Currently all packages are re-sgined on our main repository and we are inwestigating what can be a real issue here.

This is a duplicateof 344 bug.

TPG
Cooker Team

*** This bug has been marked as a duplicate of bug #344 ***
